src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -717,6 +721,10 @@ def train(self, training_params: dict = dict()):  # noqa: C901
                 -   `sg_logger_params` : dict
 
                     SGLogger parameters
+
+                -   `clip_grad_norm` : float
```

Comment:
Please add clip_grad_norm to super_gradients/recipes/training_hyperparams/default_train_params.yaml with a short doc like other training params.

Reply:
added